system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs

```diff
@@ -154,7 +154,7 @@ pub type Migrations = (
 		Runtime,
 		bridge_to_kusama_config::XcmOverBridgeHubKusamaInstance,
 	>,
-	snowbridge_pallet_system::migration::FeePerGasMigrationV0ToV1<Runtime>,
+	bridge_to_ethereum_config::migration::RemoveNativeToForeignId<Runtime>,
```

Comment:
@yrong this PR should really contain just adding one migration and nothing else :), for example:
```suggestion
	frame_support::migrations::RemoveStorage<
		"EthereumSystem", // <- define as &str param_type
		"NativeToForeignId", // <- define as &str param_type
		RocksDbWeight,
        >,
```


for example check in polkadot-sdk:
```
frame_support::migrations::RemoveStorage<
		BridgeWestendMessagesPalletName,
		OutboundLanesCongestedSignalsKey,
		RocksDbWeight,
>
```

Reply:
@bkontur The [tests](https://github.com/yrong/runtimes/blob/701ac13e4aa971f42864bd9cf74dbd955f7dc3cb/system-parachains/bridge-hubs/bridge-hub-polkadot/tests/snowbridge.rs#L430) added in this PR are meant to ensure that the PNA transfer don't break in future XCM releases (e.g. XCM V6). The reason is that the tokenID - the key of `ForeignToNativeId` storage to identify the PNA - was generated from XCM V4  and has already been finalized and stored on Ethereum side.

So we can't allow it to change. The test will fail if there is any compatalibity issue in a future release, so I prefer to keep it as a safeguard.

Meanwhile, I've simplified the migration and made some cleaned things up as you suggested in https://github.com/polkadot-fellows/runtimes/pull/730/commits/1ffb3a09544dfbd2c110f7c671eb6186299b3854.